src/wrapper.rs

```diff
@@ -8,10 +8,25 @@ use syn::{
     Type,
 };
 
-#[derive(Eq, PartialEq, Clone, Copy)]
-pub enum GenOpt {
-    All,
-    NoImplMessage,
+bitflags! {
+    pub struct GenOpt: u32 {
+        /// Generate implementation for trait `::protobuf::Message`.
+        const MSG = 0b0000_0001;
+        /// Generate getters.
+        const GET = 0b0000_0010;
+        /// Generate setters.
+        const SET = 0b0000_0100;
+        /// Generate the `new_` constructors.
+        const NEW = 0b0000_1000;
+        /// Generate `clear_*` functions.
+        const CLR = 0b0001_0000;
+        /// Generate `has_*` functions.
+        const HAS = 0b0010_0000;
+        /// Generate mutable getters.
+        const MUT = 0b0100_0000;
+        /// Generate `take_*` functions.
+        const TKE = 0b1000_0000;
+    }
 }
```

Comment:
It would be good to provide some convenience `pub const`s for `ALL`, `NO_MSG`, and the getters, setters, etc., but not `NEW`

Reply:
You mean we remove `NEW` completely?

Reply:
No I mean we should have a const called GET_SET or something defined to be `MUT | TAKE | ...`, but not including `NEW`

Reply:
`bigflags` have already created the `::all()` for you so I'll add `no_msg` and `accessor` only